tests/build.rs

```diff
@@ -42,6 +42,51 @@ fn it_builds_with_webpack_single_js() {
     cleanup(fixture);
 }
 
+#[test]
+fn it_builds_with_webpack_function_config_js() {
+    let fixture = "webpack_function_config_js";
+    create_temporary_copy(fixture);
+
+    settings! {fixture, r#"
+        type = "Webpack"
+    "#};
+
+    build(fixture);
+    assert!(fixture_out_path(fixture).join("script.js").exists());
+    assert!(fixture_out_path(fixture).join("metadata.json").exists());
```

Comment:
I would remove the metadata.json checks because it's not relevant here and I believe it has been removed in master or PRs.